precode.sh

```diff
@@ -6,38 +6,41 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir -p task/dir{1..3} task/dir3/dir4
 # изменяем текущую директорию на task
-
+cd task || exit
 # создаём пустой файл task/dir2/empty
-
+touch ./dir2/empty
 # создаём файл task/dir2/hello.sh с таким содержанием:
 # #!/bin/bash
 # echo "$1, привет!"
-
+touch ./dir2/hello.sh
+echo '#!/bin/bash
+echo "$1, привет!"' > ./dir2/hello.sh
 # устанавливаем для task/dir2/hello.sh права rwxrw-r--
-
+chmod 764 ./dir2/hello.sh
 # сохраняем список файлов task/dir2 в task/dir2/list.txt
-
+ls ./dir2 > ./dir2/list.txt
```

Comment:
Можно везде не ставить
```suggestion
ls dir2 > dir2/list.txt
```